_includes/spec-table-key.html

```diff
@@ -0,0 +1,11 @@
+
+<h5>Key to specification table</h5>
+<ul>
+  <li><em style="color: #0B794B;"><strong>Green</strong></em> properties/types are proposed by Bioschemas, or indicate proposed changes by Bioschemas to Schema.org</li>
+  <li><em style="color: #990000;"><strong>Red</strong></em> properties/types exist in the core of Schema.org</li>
+  <li><em style="color: #0000CC;"><strong>Blue</strong></em> properties/types exist in the pending area of Schema.org</li>
+  <li><em style="color: #454547;"><strong>Black</strong></em> properties/types are reused from external vocabylaries/ontologies</li>
```

Comment:
Typo: vocabylaries --> vocabularies

Reply:
Are the black ones allowed in Bioschemas profiles? I thought we wanted all the properties to be in schema.org or Bioschemas but not outside

Reply:
For properties that are going to be used by the vast majority of use cases then we should push for properties to be included into Schema.org. However, there are instances of edge cases where just using the domain term makes sense. One example is the plant community want to state the type of growth facility.